PowerQueryM/ForecastedLoad.query.pq

```diff
@@ -0,0 +1,32 @@
+let
+    MCastDomain = "demo-gas.mea-analytics.tools",
+    MaxDate = DateTime.Date(DateTime.LocalNow()),
+    MinDate = Date.AddDays(MaxDate, -60),
+    ApiKey = "your API Key here",
```

Comment:
I think we should move this to the second lines so that the first 2 lines are what you need to edit to get the example working.

Reply:
And then maybe adding a white space row after